addons/sprite-properties/userscript.js

```diff
@@ -36,11 +36,22 @@ export default async function ({ addon, console, msg }) {
     }
   }
   // Close properties panel when mouse leaves the entire sprite panel
+  let popoverAbove;
   document.body.addEventListener(
-    "mouseleave",
+    "mousemove",
     (e) => {
-      if (e.target.matches('[class*="sprite-selector_sprite-selector_2KgCX"]')) {
+      let popoverAbove = document.querySelector('[class*="Popover-above"]');
```

Comment:
`document.querySelector` is slow on large projects. perhaps you can use eg. `e.target.closest(..)` instead